utilities/backupable/backupable_db.cc

```diff
@@ -1182,7 +1182,9 @@ Status BackupEngineImpl::CreateNewBackupWithMetadata(
       new_backup->SetSequenceNumber(sequence_number);
     }
   }
-  ROCKS_LOG_INFO(options_.info_log, "add files for backup done, wait finish.");
+  ROCKS_LOG_INFO(options_.info_log,
+                 "add files for backup done (%s), wait finish.",
+                 s.ok() ? "OK" : "not OK");
```

Comment:
If `s` is not `ok`, can we print `s.ToString()` instead of "not OK"?

Reply:
It's printed below in "Backup failed ...". No need to print the details twice. I just find it very misleading to print this line when status is known not-OK. Adding OK vs. not OK can help to localize what reported the non-OK status.